src/utils/calendarEvents.ts

```diff
@@ -133,13 +182,13 @@ export const useCalendarEvents = (
             text,
             description,
             labels,
-            start: dayjs(start),
+            start: dayjs(start).minute(minutesOffset),
```

Comment:
We are losing minutes because it's set to 0.

Reply:
fixed